python/test/unit/io/test_adios2.py

```diff
@@ -291,6 +291,8 @@ def partitioner(comm, nparts, local_graph, num_ghost_nodes):
 def test_vtx_reuse_mesh(tempdir, dim, simplex, reuse):
     "Test reusage of mesh by VTXWriter"
     adios2 = pytest.importorskip("adios2")
+    if not adios2.is_built_with_mpi:
```

Comment:
Keep in mind `adios2.is_built_with_mpi` is new, i.e. not available in ADIOS2 2.8.1.  2.10.0 has it (released two weeks ago)

Reply:
Alternatively could place the test is a try block and catch the `AttributeError` from adios2 2.8 or 2.9.  Perhaps that could be more helpful for the dolfinx 0.8 release